lib/tutorials/index.js

```diff
@@ -108,6 +116,13 @@ module.exports = function makePlugin(dir) {
                             return tag.trim();
                         }) || [];
 
+                        // Add tutorial tags to list (only for english)
```

Comment:
Minor: For clarity, we should have this in a separate loop. It makes the logic clearer when reading the code that there's a distinct block that gets the tags, then another for the index rather than having intermixed

Reply:
I agree, just went in and refactored that out!